### PR TITLE
Update Dependency Track To 4.6.0

### PR DESCRIPTION
**Pull request by BlythMeister**
4.6.0 of api and front end released

I'm not sure if this is the _only_ changes required to update the helm chart.

I also noticed that the current chart was using a mismatch of api and frontend, so have corrected this

### CODE REVIEW

**Review by davidkarlsen — APPROVED** [review]
lgtm - have you tested it locally? Are other changes needed too?
I saw they now support metrics, can we enable that too - although that can come in a later PR.

**Comment by BlythMeister** [comment]
I'll be honest, I have no idea how helm charts work....my ops team said they use these charts so that's why I can't have the new version 🤣

**Comment by davidkarlsen** [comment]
> I'll be honest, I have no idea how helm charts work....my ops team said they use these charts so that's why I can't have the new version 🤣

It would be good if they could test it 1st (simply change the install values to correspond to the ones in this PR).

**Comment by BlythMeister** [comment]
I have done a small test and it does appear to work as expected.

File: charts/dependency-track/Chart.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 apiVersion: v2
-appVersion: 4.5.0
+appVersion: 4.6.0
```
Comment (davidkarlsen) [inline charts/dependency-track/Chart.yaml:2]:
```suggestion
appVersion: 4.6.1
```

**Review by davidkarlsen — APPROVED** [review]

**Comment by BlythMeister** [comment]
I wasn't sure on making app version 4.6.1.
Since frontend and server can now version different...which one is the "app version"

**Pull request merged**